aiogram/dispatcher/webhook.py

```diff
@@ -448,6 +448,17 @@ def _global_disable_web_page_preview():
             return bot.disable_web_page_preview
 
 
+class ProtectContentMixin:
```

Comment:
Why do we need a mixin? Describe a case

Reply:
for consistency with other mixins like https://github.com/aiogram/aiogram/blob/dev-2.x/aiogram/dispatcher/webhook.py#L417  and https://github.com/aiogram/aiogram/blob/4d2d81138681d730270819579f22b3a0001c43a5/aiogram/dispatcher/webhook.py#L428
Client code can invoke method protect_content on created object of Response for protecting message content